common/util.go

```diff
@@ -52,14 +52,15 @@ func ParseConnectorMetadata() (ConnectorMetadata, error) {
 }
 
 // HandleHTTPRequest sends message and headers data to HTTP endpoint using POST method and returns response on success or error in case of failure
-func HandleHTTPRequest(message string, headers http.Header, data ConnectorMetadata, logger *zap.Logger) (*http.Response, error) {
+func HandleHTTPRequest(message string, headers http.Header, data ConnectorMetadata, logger *zap.Logger) (int, *http.Response, error) {
```

Comment:
Can we use `*int` , when we have an error...we would return `nil` instead of -1 . What do you think?

Reply:
I prefer it!